stubs/simple-websocket/simple_websocket/ws.pyi

```diff
@@ -0,0 +1,132 @@
+import socket
+import threading
+from _typeshed import FileDescriptorLike
+from _typeshed.wsgi import WSGIEnvironment
+from collections.abc import Callable
+from selectors import SelectorKey, _EventMask
+from ssl import SSLContext
+from typing import Any, Protocol, type_check_only
+
+from wsproto import ConnectionType, WSConnection
+from wsproto.events import Request
+from wsproto.frame_protocol import CloseReason
+
+@type_check_only
+class _ThreadClassProtocol(Protocol):
+    name: str
+    def __init__(self, target: Callable[..., Any], *args: Any, **kwargs: Any) -> None: ...
```

Comment:
Can't we use a ParamSpec here?

Reply:
on second read - it doesn't actually need args or kwargs - the code only uses `target`